### ISSUE TITLE
It breaks Visual Studio 1.47.0 on Reload Custom CSS and JS

### ISSUE THREAD

**Issue by SoldierCorp** [be5invis/vscode-custom-css#123]
Hi!

I have Arch Linux and I just updated the Visual Studio to the latest version 1.47.0 but when I reload the css and js, is killing Visual Studio and I have to uninstall and install it again.

**Comment by supercvetic** [comment]
Same here. MX Linux. Updated VS Code, Reload CSS and JS. Result: VS Code blank window, only top menus are visible.

**Comment by be5invis** [comment]
I tested with an empty CSS and it doesn't break anything. What is the style you used and what are the Developer Tool messages (could be seen using Help → Toggle Developer Tools)?

**Comment by supercvetic** [comment]
Style used: [https://marketplace.visualstudio.com/items?itemName=webrender.synthwave-x-fluoromachine](https://marketplace.visualstudio.com/items?itemName=webrender.synthwave-x-fluoromachine)

Error messages (not sure if related, but happens as soon as I choose Reload CSS and JS and restart VS):
`﻿﻿workbench.js:188 Uncaught TypeError: require.__$__nodeRequire is not a function
    at getLazyEnv (workbench.js:188)
    at workbench.js:37
getLazyEnv @ workbench.js:188
(anonymous) @ workbench.js:37
`
On next VS Code restart, not even Toggle Developer Tools works.

**Comment by supercvetic** [comment]
Ok, this works without problem: Disable CSS and JS - Enable CSS and JS. After that cycle, Reload works. No errors in Console.

**Comment by rodmax** [comment]
Hi i have the same issue

@supercvetic I could not get your workaround. Please point me more detailed steps how you do it

My steps:
- update code to 1.47
- Run Reload custom CSS & JS
- Restart vscode
- get the issue: empty vscode window. Everything not working :cry: 

I tried disable extension via terminal (`code --disable-extension be5invis.vscode-custom-css`) but with no success

Right now my workaround is **reinstall vscode and not reload custom-css extension**

Thanks

vscode 1.47
OS ubunutu 20.04

**Comment by supercvetic** [comment]
@rodmax My workaround is: Reinstall VS Code, Disable CSS and JS, Enable CSS and JS. Dont run Reload custom CSS and JS.

**Comment by PierreBergamin** [comment]
Having the same issue on Ubuntu 20.04 and vscode 1.47.3.

**Comment by rodmax** [comment]
still happening in 1.48
I open debug console before run "reload custom css and js" and see the error:

![image](https://user-images.githubusercontent.com/6079597/91810437-8a3f1300-ec36-11ea-9efe-3a6b3635b319.png)


In my case i have only two options

```json
"vscode_custom_css.imports": ["file:///home/max/.config/Code/User/custom-css/index.css"],
"vscode_custom_css.policy": true,
```


@PeterMortensen  above i describe not quite a workaround but vscode "recovery"

step 1.
```bash
sudo apt remove code && sudo apt install code
```

step 2.
DO NOT perform `Reload custom CSS and Js`

So you make vscode working but without a CSS & JS customizations at all

**Comment by be5invis** [comment]
Sorry but I am not able to reproduce this (at least on my machine). I tried to add this extension and load one custom CSS and nothing bad happened.
Maybe some plugins are conflicting?

**Comment by rodmax** [comment]
@be5invis thanks for reply. I will try to remove all extensions and clean reinstall vscode and your extension today evening. And notify you about results

**Comment by rodmax** [comment]
@be5invis it seems i catch reason of error

### TL;DR; to fix issue without fix extension

```bash
sudo apt remove code
rm /usr/share/code/resources/app/out/vs/code/electron-browser/workbench/workbench.bak-customcss
sudo apt install code
sudo chown -R $(whoami) /usr/share/code  # this step may be not needed
code 
# run vs-code & apply "Reload custom css & js". it seems to bee working
```

### (i believe) reason of error

as i can see from sources, `extension.updateCustomCSS`(*Reload Custom CSS and JS*) do follow:
- `fUpdate()`
  - `fUninstall(willReinstall=true)`
    -  `restoreBak(willReinstall=true)` :beetle: bug here!!!
![image](https://user-images.githubusercontent.com/6079597/93934820-5f734680-fd2c-11ea-9f28-bfbc3ac5bf8f.png)
- `fInstall()` custom css applied to old (<v.1.47) workbanch.html


### How to reproduce
```bash
# Remove vscode & backup
sudo apt remove code
rm /usr/share/code/resources/app/out/vs/code/electron-browser/workbench/workbench.bak-customcss
 
# install code version < 1.47 for example
sudo apt install code=1.46.1-1592428892
sudo chown -R $(whoami) /usr/share/code  # JUST in case

# Run vscode and apply customs CSS & JS
code 
# After that workbench.bak-customcss will be created with old < 1.47 version of workbench.html

# Then install modern vesion of vscode
sudo apt install

# Run code and run "Reload Customs CSS & JS
# During this step extension restore workbench.html with old version from backup and IT BREAKS code (!!!!)
```


<details>
<summary>workbench.html v1.46</summary>

```html
<!-- Copyright (C) Microsoft Corporation. All rights reserved. -->
<!DOCTYPE html>
<html>
→   <head>
→   →   <meta charset="utf-8" />
→   →   <meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src 'self' https: data: blob: vscode-remote-resource:; media-src 'none'; frame-src 'self' https://*.vscode-webview-test.com; object-src 'self'; script-src 'self' 'unsafe-eval'; style-src 'self' 'unsafe-inline'; connect-src 'self' https:; font-src 'self' https: vscode-remote-resource:;">
→   </head>
→   <body aria-label="">
→   </body>

→   <!-- Startup via workbench.js -->
→   <script src="workbench.js"></script>
</html>

```
</details>

<details>
<summary>workbench.html v1.49 (as you can see three new `<script>` add which will be erased after restore)</summary>

```html
<!-- Copyright (C) Microsoft Corporation. All rights reserved. -->
<!DOCTYPE html>
<html>
→   <head>
→   →   <meta charset="utf-8" />
→   →   <meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src 'self' https: data: blob: vscode-remote-resource:; media-src 'none'; frame-src 'self' vscode-webview: https://*.vscode-webview-test.com; object-src 'self'; script-src 'self' 'unsafe-eval'; style-src 'self' 'unsafe-inline'; connect-src 'self' https:; font-src 'self' https: vscode-remote-resource:;">
→   </head>
→   <body aria-label="">
→   </body>

→   <!-- Init Bootstrap Helpers -->
→   <script src="../../../../bootstrap.js"></script>
→   <script src="../../../../vs/loader.js"></script>
→   <script src="../../../../bootstrap-window.js"></script>

→   <!-- Startup via workbench.js -->
→   <script src="workbench.js"></script>
</html>

```
</details>
